lib/dependencies.js

```diff
@@ -11,11 +11,14 @@ container.provide("config", require("../config.json"));
 // Our code, written in a way that it expects the container to be passed
 // into a registration function.
 container.instance("apiServer", require("./api-server"));
+container.singleton("httpsConfig", require("./https-config"));
```

Comment:
Unless there is a reason to use a singleton, this should use `.instance()` instead.